CodeForAfricaLabs/CitizenReporter.Android
# Ft fix camera orientation

## PR Status
**READY**

## Story ID
The unique identifier of the story/stories worked upon as shown on trello.


## Description
On change of device orientation, say from Portrait to Landscape, the icons are given a rotation animation as they take on their new rotation angles.
Save and restore camera state on change of orientation.
I also added overlay SVG drawable assets for the landscape mode. The assets used in the portrait form don't scale to fit in the landscape mode.
Added a black background to the image preview activity.

## New Packages Added
* None

## Todos
- [x] Tested and working on development environment
- [ ] Reviewed and approved by a senior dev

## Steps to Test or Reproduce
* `git checkout ft-fix-camera-orientation`
* Build the Android App.
* Rotate the device orientation and see the icons animated.

```sh
git pull --prune
git checkout <ft-fix-camera-orientation>
```

## Impacted Areas in Application
Components of the application that this PR will affect:
* camera
* Android Manifest
* Drawables

## Known Issues
* The camera is a bit slow.
